### DISCUSSION TITLE
CUSA09473_01.01.json Resident Evil 7 Biohazard Gold Edition

### DISCUSSION THREAD

**Discussion by skx7** [GoldHEN/GoldHEN_Cheat_Repository#78] [Cheat requests]
Is someone able to confirm if these cheats actually work? God Mode cheat doesn't have any affect. Infinite Ammo even crashes the game upon activating...

**Comment by kmeps4** [comment]
Please go to https://github.com/GoldHEN/GoldHEN_Cheat_Repository/discussions/42

**Comment by abdiziriel** [comment]
Question on Village gold. Cheats available are infinite ammo, craft, healt and pipe bomb. They used to work but now only ammo and pipe bomb works. 

Not sure what went wrong
